src/site/_includes/course.njk

```diff
@@ -1,5 +1,7 @@
 ---
 layout: base
+tags:
+  - course
```

Comment:
should this be removed? it doesn't seem to do anything

Reply:
I though of using it for search faceting, but its not needed for now.